raspbian/updating.md

```diff
@@ -23,7 +23,7 @@ sudo apt update
 Next, **upgrade** all your installed packages to their latest versions with the following command:
 
 ```bash
-sudo apt dist-upgrade
+sudo apt full-upgrade
 ```
 
 Note that `dist-upgrade` is used in preference to a simple `upgrade`, as it also picks up any dependency changes that may have been made. 
```

Comment:
I guess this line should have been changed to `full-upgrade` too? :eyes:

Reply:
No worries, I only spotted it on a proof read before merging!